homeassistant/components/blebox/config_flow.py

```diff
@@ -169,6 +182,10 @@ async def async_step_user(self, user_input=None):
             return self.handle_step_exception(
                 "user", ex, schema, *addr, UNSUPPORTED_VERSION, _LOGGER.debug
             )
+        except UnauthorizedRequest as ex:
+            return self.handle_step_exception(
+                "user", ex, schema, *addr, CANNOT_CONNECT, _LOGGER.error
```

Comment:
We normally use `invalid_auth` error string when authentication fails.

https://github.com/home-assistant/core/blob/ddfad75eb7a6807a8b40490650540bca7e4e23b1/homeassistant/strings.json#L108